webserver/static/robots.txt

```diff
@@ -2,3 +2,5 @@ User-agent: *
 Disallow: /current-status
 Disallow: /user/
 Disallow: /1/
+Disallow: /api
```

Comment:
Is there a reason for using `/api` here and not `/api/`? (I'm a bit rusty on my robots.txt syntax.)